### ISSUE TITLE
Langues d'oïl classification

### ISSUE THREAD

**Issue by mvergezc** [glottolog/glottolog#1082]
Hello ! 
Here are some thoughts on the classification of "Langues d'oïl". 
I'm mainly concerned with Poitevin-Saintongeais, which should be described as a language with two variants, Poitevin and Saintongeais. Poitevin-Saintongeais belongs to the Langues d'Oïl family, along with Picard (which is now directly included in Central Oïl).
French should also be considered a Langue d'oïl.
Bourguignon should be written as Bourguignon-Morvandiau.
Lorraine should be written as Lorrain roman.
The list of Langues d'oïl proposed here is given in Eloy & Jagueneau (2013), " Les langues d'oïl " dans Kremnitz, G. (ed) " Histoire sociales de langues de France ", Presses Universitaires de Rennes.
Angevin, Berrichon, Bourbonnais (currently included in French) together with Tourangeau, Manceau, Mayennais, Percheron. They are better described as Langues d'oïl. However, their classification is more controversial.

>Oïl
>>Central oïl
>>>Langues d’oïl
>>>>Bourguignon-morvandiau
>>>>Champenois
>>>>Franc-Comtois
>>>>French
>>>>Gallo
>>>>Lorrain roman 
>>>>Normand
>>>>Picard
>>>>Wallon
>>>>Poitevin-Saintongeais
>>>>>Poitevin
>>>>>Saintongeais

**Comment by d97hah** [comment]
The source you mention does not attempt to give a historical clf of the
lects involved. But I can change the names to Bourguignon-Morvandiau and
Lorrain Romann and make a Poitevin-Saintongeais sub-node. all the best, H

Pada Sel, 18 Jun 2024 pukul 09.37 mvergezc ***@***.***>
menulis:

**Comment by mvergezc** [comment]
Thank you for your reply. It's a good start. However, I'm still bothered by the fact that "Langues d'oïl" are subsumed under "French". Many sources will confirm that "French" is an oïl language, except that it has gained full recognition. Henriette Walter says: "Il faut donc bien comprendre que non seulement les patois ne sont pas du français déformé, mais que le français n'est qu'un patois qui has réussi. "It is therefore important to understand that not only are patois not a deformed French, but that French is only a successful patois. You can have a look at her book "Aventures et mésaventures des langues de France" published by Honoré Champion. In the meantime, I will look at other references that would give you an historical perspective. Thanks a lot, Marianne

**Comment by d97hah** [comment]
It depends on what you want to understand by the label French. You appear
to mean Standard French and Glottolog something else. For sure one could
have Francien, Standard French and many other varieties of French included
but Glottolog does not aim to be complete wrt to dialects, so while I can
add some more, there will be other things missing. In addition many
conventional dialect classifications are not (and are not meant to be)
historical (incl that of H Walter if I am not misremembering) so some work
is needed to adapt them to the goals of Glottolog. all the best, H

Pada Rab, 19 Jun 2024 pukul 17.17 mvergezc ***@***.***>
menulis:

**Comment by mvergezc** [comment]
Thank you for the explanation. I now understand the historical perspective. Although I have a thousand questions, I don't know if this is the right place. Could you just tell me when the change to the Poitevin-Saintongeais sub-node will come into effect? All the best, Marianne

**Comment by xrotwang** [comment]
> Could you just tell me when the change to the Poitevin-Saintongeais sub-node will come into effect?

We release Glottolog about twice a year. Last release was in March, so I'd say sometime in fall we'll release Glottolog 5.1 and update the website at https://glottolog.org

**Comment by xrotwang** [comment]
To be released with Glottolog 5.1 in the coming week:
```
$ glottolog tree lang1337
INFO    glottolog/glottolog at /home/robert/projects/glottolog/glottolog
Indo-European [indo1319]
   └─ Classical Indo-European [clas1257]
      └─ Italic [ital1284]
         └─ Latino-Faliscan [lati1262]
            └─ Latinic [lati1263]
               └─ Imperial Latin [impe1234]
                  └─ Romance [roma1334]
                     └─ Italo-Western Romance [ital1285]
                        └─ Western Romance [west2813]
                           └─ Shifted Western Romance [shif1234]
                              └─ Northwestern Shifted Romance [nort3208]
                                 └─ Gallo-Rhaetian [gall1280]
                                    └─ Oil [oila1234]
                                       └─ Central Oil [cent2283]
                                          └─ Macro-French [macr1273]
                                             └─ Global French [glob1239]
                                                └─ French [stan1290]
                                                   └─ Langues d'Oil [lang1337]
                                                      ├─ Bourguignon-Morvandiau [bour1247]
                                                      ├─ Champenois [cham1332]
                                                      ├─ Franc-Comtois [fran1270]
                                                      ├─ Francien [fran1273]
                                                      ├─ Gallo [gall1275]
                                                      ├─ Lorrain Roman [lorr1242]
                                                      ├─ Normand [norm1245]
                                                      └─ Poitevin-Saintongeais [poit1241]
                                                         ├─ Poitevin [poit1240]
                                                         └─ Saintongeais [sant1407]
```
